src/strategies.js

```diff
@@ -1,15 +1,16 @@
-import initiationStrategies from './initiationStrategies';
-
-export const strategies = initiationStrategies({
-  nextTick(pluginFn, $$) {
-    return window.setTimeout(() => pluginFn(), 0);
-  },
-
-  immediate(pluginFn, $$) {
-    return pluginFn() || {};
-  },
-
-  hover(pluginFn, $$) {
-    return $$.one('mouseover', () => pluginFn());
-  }
-});
+export function nextTick(pluginFn, _el) {
+  return window.setTimeout(() => pluginFn(), 0);
+}
+export function immediate(pluginFn, _el) {
+  return pluginFn() || {};
+}
+export function hover(pluginFn, el) {
+  const isJqueryEl = el.jquery !== undefined;
```

Comment:
Can we just leave it up to the the client code to add a new hover strategy with `widget.strategies.add('hover', () => {})`, if they want to have a jQuery hover option?

Reply:
In fact I don't think this works, since #31 the hover strategy has been broken.

Notice that in UE we're using `1.3.0`. We use `2.0.0` in another project and haven't used the hover strategy in that project yet.

Reply:
Got it, still keeping the registry and opening it up to client registering new strategies makes sense.